kubehub/migrations/0002_auto_20200128_1132.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.2 on 2020-01-28 11:32
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kubehub', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='VmGroups',
```

Comment:
should not be plural, it's VMGroup not VMGroups

Reply:
fixed in 7f1c04a